README.md

```diff
@@ -474,6 +474,7 @@ These modules are parts of textlint.
 | [`textlint-formatter`](/packages/textlint-formatter) | [![npm](https://img.shields.io/npm/v/textlint-formatter.svg?style=flat-square)](https://www.npmjs.com/package/textlint-formatter) | textlint output formatter |
 | [`textlint-plugin-markdown`](/packages/textlint-plugin-markdown) | [![npm](https://img.shields.io/npm/v/textlint-plugin-markdown.svg?style=flat-square)](https://www.npmjs.com/package/textlint-plugin-markdown) | markdown support for textlint |
 | [`textlint-plugin-text`](/packages/textlint-plugin-text) | [![npm](https://img.shields.io/npm/v/textlint-plugin-text.svg?style=flat-square)](https://www.npmjs.com/package/textlint-plugin-text) | plain text support for textlint |
+| [`textlint-plugin-html`](/packages/textlint-plugin-html) | [![npm](https://img.shields.io/npm/v/textlint-plugin-html.svg?style=flat-square)](https://www.npmjs.com/package/textlint-plugin-html) | HTML support for textlint |
```

Comment:
It is not a core package. 
`textlint-plugin-html` is not built-in plugin in textlint.
It is a reason that I've noted "secondary" at #270 

I wonder what to import this plugin into monorepo.
Because, I'm not a html plugin user, and `textlint-plugin-html` is not stable plugin https://github.com/textlint/textlint-plugin-html/issues/2 (version < 1.0.0)

If core maintainer exists for html plugin, I'm welcome to import `textlint-plugin-html` into repository.

@kemsakurai Are you interested in maintaining `textlint-plugin-html`?

Reply:
Oops, I make a mistake.
I'm sorry for submit PR without sufficient understanding this repository.

Reply:
@Leko Don't worry. It's my mistake.